src/reducers/index.js

```diff
@@ -9,12 +9,21 @@ const initialState = {
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
+
 export default (state = initialState, action) => {
   switch (action.type) {
+
     case INCREMENT:
     // Fill in the body of this case
+
+    return {...state, count: initialState.count++}
```

Comment:
Same thing, but `initialState` has been reassigned to `state` so it's better to use it
line 13: `export default (state = initialState, action) =>`
```suggestion
    return {...state, count: state.count++}
```